plugins/archiving/index.js

```diff
@@ -67,6 +67,7 @@ module.exports = (app, config, redis, ot) => {
       }
       otSDK.startArchive(sessionId, {
         name: room,
+        resolution: '1280x720',
```

Comment:
extra trailing comma

Reply:
Yeah, that's intentional. We use the airbnb style guide and they require it now. It's strange, they used to require that you don't have it, now they require that you do.

Reply:
https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8